gui/default/syncthing/core/syncthingController.js

```diff
@@ -371,6 +377,9 @@ angular.module('syncthing.core')
             $scope.config.options._globalAnnounceServersStr = $scope.config.options.globalAnnounceServers.join(', ');
             $scope.config.options._urAcceptedStr = "" + $scope.config.options.urAccepted;
 
+            $scope.config.gui.enableNotifications = PushNotifications.isEnabled();
+            $scope.config.gui.supportNotifications = PushNotifications.isSupported();
```

Comment:
These are virtual fields, not actually stored in the config, hence probably should follow the unwritten "start not real stuff with `_` convention"